  armpap:
    # Publish all joints states
    joint1_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

      
    # Position Controllers
    motor_base_joint_controller:
      type: effort_controllers/JointPositionController
      joint: motor_base_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}
    
    link_right01_joint_controller:
      type: effort_controllers/JointPositionController
      joint: link_right01_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}

    link_left01_joint_controller:
      type: effort_controllers/JointPositionController
      joint: link_left01_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}

    link_left02_joint_controller:
      type: effort_controllers/JointPositionController
      joint: link_left02_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}
    
    gripper_base_joint_controller:
      type: effort_controllers/JointPositionController
      joint: gripper_base_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}

    gripper_left_joint_controller:
      type: effort_controllers/JointPositionController
      joint: gripper_left_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}
    
    gripper_right_joint_controller:
      type: effort_controllers/JointPositionController
      joint: gripper_right_joint
      pid: {p: 100.0, i: 0.01, d: 10.0}
